snuba/state/cache/redis/backend.py

```diff
@@ -297,9 +334,15 @@ def get_readthrough(
             return function()
 
         try:
-            return self.__get_readthrough(
-                key, function, record_cache_hit_type, timeout, timer
-            )
+            # set disable_lua_scripts to use the simple read-through cache without queueing.
+            if get_config("read_through_cache.disable_lua_scripts", 0):
```

Comment:
I would change this to a percentage check, so you can roll out incrementally.

```
if random.random() < get_config(...):
```

Reply:
I will update the code.